package.json

```diff
@@ -24,11 +24,12 @@
   "author": "Daniel Durante <me@danieldurante.com>",
   "license": "MIT",
   "dependencies": {
-    "auth-net-request": "2.2.2",
+    "auth-net-request": "SC5/authorize-net-request",
```

Comment:
Will the new package of auth-net-request be 2.2.3? The change won't work with the 2.2.2 package.

-Mikael

> Daniel Durante <notifications@github.com> kirjoitti 25.8.2015 kello 20.28:
> 
> In package.json:
> 
> > @@ -24,11 +24,12 @@
> >    "author": "Daniel Durante <me@danieldurante.com>",
> >    "license": "MIT",
> >    "dependencies": {
> > -    "auth-net-request": "2.2.2",
> > +    "auth-net-request": "SC5/authorize-net-request",
> Can you revert this line please?
> 
> —
> Reply to this email directly or view it on GitHub.
>